include/swift/SILOptimizer/Analysis/EpilogueARCAnalysis.h

```diff
@@ -247,14 +247,17 @@ class EpilogueARCFunctionInfo {
     if (Iter != ARCCache.end())
       return Iter->second;
 
-    EpilogueARCContext CM(Kind, Arg, F, PO->get(F), AA, RC->get(F));
     // Initialize and run the data flow. Clear the epilogue arc instructions if the
     // data flow is aborted in middle.
-   if (!CM.run()) { 
-     CM.resetEpilogueARCInsts();
-     return CM.getEpilogueARCInsts();
+    if (!Context.run(Kind, Arg)) {
+      Context.reset();
+      return llvm::SmallSetVector<SILInstruction *, 1>();
```

Comment:
I am fine making that change. But I would like to do it on master, not here. How does that sound?